source/_integrations/vodafone_station.markdown

```diff
@@ -34,13 +34,38 @@ There is support for the following platform types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+{% configuration %}
+  host:
+    required: true
+    description: The IP address of the Vodafone Station router.
+    type: string
+  username:
+    required: true
+    description: The username of the Vodafone Station router.
+    type: string
+  password:
+    required: true
+    description: The password of the Vodafone Station router.
+    type: string
+{% endconfiguration %}
+
 ## Integration options
 
 It is possible to change some behaviors through the {% term integration %} options.
 To change the settings, go to {% my integrations title="**Settings** > **Devices & services**" %}. On the **Vodafone Station** integration, select the cogwheel. Then select **Configure**.
 
 - **Consider home**: Number of seconds that must elapse before considering a disconnected device "not at home".
 
+## Removing the integration
```

Comment:
I would move the removal section to the end of the docs as the task is done at the end of the integration life cycle.

Reply:
Done